banyan/resource_accesstier_group_test.go

```diff
@@ -0,0 +1,46 @@
+package banyan
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccAccessTierGroup_basic(t *testing.T) {
+
+	rName := fmt.Sprintf("tf-acc-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+
+	resource.Test(t, resource.TestCase{
+		Providers:    testAccProviders,
+		CheckDestroy: nil,
+		Steps: []resource.TestStep{
+			// Creates the access_tier_group with the given terraform configuration and asserts that the access_tier_group is created
+			{
+				Config: testAccAccessTierGroup_basic_create(rName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("banyan_accesstier_group.example", "name", rName),
+				),
+			},
+		},
+	})
+}
+
+func testAccAccessTierGroup_basic_create(name string) string {
+	return fmt.Sprintf(`
+resource "banyan_accesstier_group" "example" {
+	name                   = "%s"
+	description            = "testing-1"
+	cluster                = "cluster1"
+	dns_search_domains     = ""
+	advanced_settings      = "{\"kind\":\"BanyanAccessTierLocalConfig\",\"api_version\":\"rbac.banyanops.com/v1\",\"type\":\"attribute-based\",\"metadata\":{},\"spec\":{\"base\":{\"shield_address\":\"ted-us-west1.shield.bnntest.com:34140\",\"site_address\":\"\"},\"logging\":{},\"events\":{},\"hosted_web_services\":{}}}"
```

Comment:
@rohitSangamnerkar this is not a right way for advance_settings, we could reuse LocalConfig from accessTier and could be key value pairs. Could you create another PR for the same ?